lib/rules/id-length.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @fileoverview Rule that warns when identifier names are shorter or longer
+ *               than the values provided in configuration.
+ * @author Burak Yigit Kaya aka BYK
```

Comment:
I mean I want anyone to do whatever he or she wants with this code and a copyright implies that they need to consult to me or something right? That's why I'm resisting. If my understanding is not correct though, I'd add it especially for consistency :)